src/dialogs/settingsdialog.h

```diff
@@ -48,6 +48,7 @@ Q_OBJECT
         ToolbarPage,
         DebugOptionPage,
         EditorPage,
+        PanelsPage,
```

Comment:
I think the order in an enum does not matter; I put it there because in the QT Creators Object Tree the panelsPage appears before the gitPage and so both are aligned.